finagle/h2/src/main/scala/com/twitter/finagle/buoyant/H2.scala

```diff
@@ -18,7 +19,14 @@ object H2 extends Client[Request, Response] with Server[Request, Response] {
   private[this]type Http2FrameTransport = Transport[Http2Frame, Http2Frame]
 
   object Client {
-    val newStack: Stack[ServiceFactory[Request, Response]] = StackClient.newStack
+    val newStackWithoutTlsClientPrep: Stack[ServiceFactory[Request, Response]] =
+      StackClient.newStack[Request, Response]
+
+    val newStack: Stack[ServiceFactory[Request, Response]] = {
```

Comment:
newStack is used in finagle...H2.Client(), newStackWithoutTlsClientPrep is used in router.H2.clientStack